test/binaries/test-libexif-0.6.21.c

```diff
@@ -0,0 +1,12 @@
+#include <stdio.h>
+
+int main() {
+	printf("This program is designed to test the cve-bin-tool checker.");
+	printf("It outputs a few strings normally associated with libexif 0.6.21");
+	printf("They appear below this line.");
+	printf("------------------");
+	printf("libexif-12.mo");
+	printf("The EXIF library allows you to parse an EXIF file and read the data from those tags.");
```

Comment:
This looks like a test that didn't work out and should also be removed since it's irrelevant to this PR.